AirBench/Scripts/Home.js

```diff
@@ -60,6 +72,36 @@
     });
     map.on('click', clickMarker);
     //map.addEventListener('click', clickMarker);
+    function updateList(){
+        let min = document.getElementById('min').value;
+        let max = document.getElementById('max').value;
+        console.log(min, max);
+        //console.log(min, max);
+        //call api
+        //let response = await fetch("/api/bench/filtered/?min=" + min + "&max=" + max);
+        //let jsonResult = await response.json();
+        //console.log(jsonResult);
+
+        //filter table rows based on data -> id
+        let table = document.getElementById("list");
+        let rows = table.getElementsByTagName("tr");
```

Comment:
You could alternatively use `document.querySelectorAll('#list tr')` to get the collection of `tr` elements.